website/src/data/users.tsx

```diff
@@ -69,7 +69,7 @@ export type User = {
 
 // LIST OF AVAILABLE TAGS
 // Available tags to assign to your site
-// Please choose widely, we'll remove unappropriate tags
+// Please choose wisely, we'll remove unappropriate tags
```

Comment:
Hi @sohamsshah seems you haven't granted me write access. I suggest you change this to:

```
// Please choose all tags that you think might apply.
// We'll remove inappropriate tags, but it's less likely that we add tags.
```